### DISCUSSION TITLE
Particles Update

### DISCUSSION THREAD

**Discussion by mare-on** [Zrips/CMI#4717] [Feature Requests]
Hi,
I would like to use "**cmi panimation**" but I found one problem with design.

Particles in Vanilla Mineecraft are nice with random spreading but this command uses only circle type which is not the best looking effect in some times. I explored every option and I didn't found anything useful which could help.

CMI is really great system so this missing feature should be implemented because hey... only circle effect? It's shame.

**Comment by Zrips** [comment]
In general panimation was created for internal use only, i only give access to it throw command. Circle is limited and new type of effect could be added if i would only have basic idea how new effects should actually behave. Like variable types and behavior from them
